src/main/java/net/aufdemrand/denizen/events/player/PlayerShearsScriptEvent.java

```diff
@@ -90,11 +90,11 @@ public ScriptEntryData getScriptEntryData() {
     }
 
     @Override
-    public HashMap<String, dObject> getContext() {
-        HashMap<String, dObject> context = super.getContext();
-        context.put("entity", entity);
-        context.put("state", entity); //DEPRECATED because...dumb
-        return context;
+    public dObject getContext(String name) {
+        if (name.equals("entity")) {
+            return entity;
+        }
+        return super.getContext(name);

```

Comment:
You still need to include 'state' in here. Just cause it is Deprecated doesn't mean players might not use it.